packages/phisherman/src/register.ts

```diff
@@ -0,0 +1,11 @@
+import { container, Plugin, preGenericsInitialization, SapphireClient } from '@sapphire/framework';
+import type { ClientOptions } from 'discord.js';
+import { Phisherman } from '.';
```

Comment:
```suggestion
import { Phisherman } from './lib/Phisherman';
import './index';
```

The bottom part to ensure that calling `/register` also includes the module augmentations.